libraries/rush-lib/src/logic/operations/IPCOperationRunnerPlugin.ts

```diff
@@ -51,12 +55,20 @@ export class IPCOperationRunnerPlugin implements IPhasedCommandPlugin {
             continue;
           }
 
-          const rawScript: string | undefined = project.packageJson.scripts?.[`${phase.name}:ipc`];
+          const { name: phaseName } = phase;
+
+          const { scripts } = project.packageJson;
+          const rawScript: string | undefined = [
+            !isInitial && scripts?.[`${phaseName}:incremental:ipc`],
+            scripts?.[`${phaseName}:ipc`]
+          ].find((x): x is string => typeof x === 'string');
```

Comment:
```suggestion
          const rawScript: string | undefined = !isInitial && scripts?.[`${phaseName}:incremental:ipc`] ?? scripts?.[`${phaseName}:ipc`];
```

Reply:
The reason I was using `.find` is that it will return either a result that satisfies the condition or else `undefined`. This lets me put values of type `null` or `boolean` or such in the inputs and still guarantee that the result is `string | undefined`. That said I can put this back into a helper function or at least name the `(x): x is string => typeof x === 'string'` helper.

Reply:
Reverted to using `??`.